experimenter/experimenter/nimbus-ui/src/components/PageHome/SearchBar/index.tsx

```diff
@@ -66,7 +76,7 @@ const SearchBar: React.FunctionComponent<SearchBarProps> = ({
       setClearIcon(true);
 
       const newTimer = setTimeout(() => {
-        const results = fuse.search(searchTerms);
+        const results = fuse.search(inputValue);
```

Comment:
then you don't have to do this

Reply:
Yes I have already tried that, but the when you type anything it will add space after every character and show that in search bar and you could not delete anything in the search bar, you can only add new characters but they will also be added with space.
see this screen shot for reference:
![Screenshot from 2023-03-25 00-20-59](https://user-images.githubusercontent.com/108016683/227614921-a94fd445-f025-4ade-bc69-6c630c009ffe.png)
if we enter 'lol' it will be take input as 'l o l ' like this that's why I am not modifying setSeachTerms.
@yashikakhurana